pywhy_graphs/algorithms/generic.py

```diff
@@ -604,3 +606,85 @@ def inducing_path(G, node_x: Node, node_y: Node, L: Set = None, S: Set = None):
                 break
 
     return (path_exists, path)
+
+
+def find_adc(G):
+
+    """Finds an Almost Directed Cycles in a mixed edge graph.
```

Comment:
If you name it `find_*`, it should return the actual path found: E.g. see similar function in networkx: https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.cycles.find_cycle.html

Otw, you can name it `has_almost_directed_cycle`, which makes more sense. Or you can return the actual path that is the "almost directed cycle".

Reply:
Got it.